src/modules/folder/folder.domain.ts

```diff
@@ -34,6 +34,7 @@ export class Folder implements FolderAttributes {
   updatedAt: Date;
   size: number;
   sharings?: Sharing[];
+  depth: number | null;
```

Comment:
Could you please modify this type to be `depth?: number` instead? This would prevent us from adding `depth: null` to every instance where Folder.build({...values}) is used. This field will be automatically populated by the database triggers, so having it as undefined or null has the same effect unless we are specifically comparing undefined and null.

I bring this up because, for instance, you had to modify some unit tests due to this type, and I would also need to add many depth: null entries in the workspaces branch.